Akka.Serialization.MessagePack/MsgPackSerializer.cs

```diff
@@ -5,31 +5,26 @@
 //-----------------------------------------------------------------------
 
 using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
 using System.Threading;
 using Akka.Actor;
 using Akka.Configuration;
 using Akka.Serialization.MessagePack.Resolvers;
 using MessagePack;
+using MessagePack.Formatters;
 using MessagePack.ImmutableCollection;
 using MessagePack.Resolvers;
+using Newtonsoft.Json;
 
 namespace Akka.Serialization.MessagePack
 {
     public sealed class MsgPackSerializer : Serializer
```

Comment:
So this is only for polymorphic serialization, correct? No applicability for using MsgPack's schema-driven format with this serializer?

Reply:
@Aaronontheweb.

[Based on this](https://github.com/neuecc/MessagePack-CSharp#reserved-extension-types) we don't have a lot of extended types to play with. But you totally *can* within those limitations.